kustomize/move2kube/README.md

```diff
@@ -0,0 +1,78 @@
+## Prerequisites
+* The manifests are generated without the `namespace` configuration
+* Manifest names reflect the `resources` section of [kustomization.yaml](./base/kustomization.yaml)
+
+A brief explanation of the system architecture is provided in the related [document](./move2kube.md).
+
+## Kustomization options
+### Image update
+Run the following to update the default images to a custom configuration
+```
+cd base && kustomize edit set image serverless-workflow-move2kube=quay.io/orchestrator/serverless-workflow-move2kube:1234 && cd ..
+cd base && kustomize edit set image serverless-workflow-m2k-kfunc=quay.io/orchestrator/serverless-workflow-m2k-kfunc:1234 && cd ..
+```
+
+## Configure properties
+Edit configuration in [replace-config.yaml](./base/replace-config.yaml), [config.properties](./base/config.properties) and 
+[secret.properties](./base/secret.properties) to match your environment configuration.
+
+## Deploy to the cluster
+### Deploy the production environment
+This environment applies the generated manifests with minimal customizations to:
+* Use the latest application images
+* Force the `prod` profile for the SonataFlow instance
+* Deploy by default on the `sonataflow-infra` namespace
+* Mount the configurations defined in [Configure properties](#configure-properties) as environment variables
+
+The default namespace can be customized with:
+```bash
+TARGET_NS=YOUR-NS
+cd base && kustomize edit set namespace=$TARGET_NS && cd ..
+```
+
+Once the configuration is set, apply the deployment to the configured namespace with:
+```bash
+kustomize build base | oc apply -f -
+```
+
+You can monitor the deployment status with:
+```bash
+oc get sonataflow m2k -n sonataflow-infra -owide
+```
+
+And finally view the logs with:
+```bash
+oc logs -f -n sonataflow-infra -l app=m2k
```

Comment:
same as above, the namespace should be parameterized too (`${TARGET_NS}` instead of `sonataflow-infra`)